src/components/Planetarium/planetarium.css

```diff
@@ -2,6 +2,6 @@
   display: flex;
   flex-wrap: wrap;
   justify-content: space-evenly;
-  margin: 3% 0;
```

Comment:
May want to switch margins to a rigid dimension (px?)

Reply:
Yeah sure, never really sure when to use px vs. %. I'll change it to px!